package.json

```diff
@@ -1,61 +1,64 @@
 {
   "name": "@metamask/detect-provider",
   "version": "2.0.0",
-  "description": "A tiny utility for detecting the MetaMask Ethereum provider, or any EIP 1193-compliant provider.",
-  "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
-  "files": [
-    "dist/"
-  ],
-  "engines": {
-    "node": ">=14.0.0"
-  },
-  "scripts": {
-    "pretest": "yarn build",
-    "test": "node test/*",
-    "lint": "eslint . --ext ts,js,json",
-    "lint:fix": "yarn lint --fix",
-    "build": "./build.sh",
-    "setup": "yarn install && yarn allow-scripts",
-    "prepublishOnly": "yarn build && yarn test && yarn lint"
-  },
+  "description": "A tiny utility for detecting the MetaMask Ethereum provider or any EIP-1193-compliant provider.",
   "repository": {
     "type": "git",
     "url": "https://github.com/MetaMask/detect-provider.git"
   },
-  "author": "Erik Marks <rekmarks@protonmail.com>",
   "license": "ISC",
-  "private": false,
-  "publishConfig": {
-    "access": "public"
-  },
-  "keywords": [
-    "ethereum",
-    "provider",
-    "metamask",
-    "web3"
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/"
   ],
-  "bugs": {
-    "url": "https://github.com/MetaMask/detect-provider/issues"
+  "scripts": {
+    "build": "./build.sh",
```

Comment:
I opted to keep the `build` script the same, but in a future PR, I think we could just use `tsc` in a future PR like we do in the module template. That would raise the question on how to use this library in a Node context, but I really wonder whether that makes sense.